api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceService.java

```diff
@@ -40,6 +41,9 @@
 
   DbWorkspace saveWithLastModified(DbWorkspace workspace);
 
+  void requireActiveBilling(String workspaceNamespace, String workspaceId)
```

Comment:
nit: I've been naming these kind of functions starting with `validate`, which I associate with a check that throws. As written, it's active voice, as if we're telling the class to require active billing on these arguments.